src/content/docs/fr/recipes/analyze-bundle-size.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Analyzer les tailles des Bundle
+description: Apprenez à analyser le bundle généré par Astro en utilisant `rollup-plugin-visualizer`.
+i18nReady: true
+type: recipe
+---
+
+import { Steps } from '@astrojs/starlight/components';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+
+Il est important de comprendre ce qui constitue une partie d'un paquet Astro pour améliorer les performances du site. La visualisation du paquet peut donner des indications sur les modifications à apporter à votre projet pour réduire la taille du paquet.
```

Comment:
In the title of the page, in the description and when you talk about Rollup, you use "bundle". I think we should leave it here too to be consistent.
```suggestion
Il est important de comprendre ce qui fait partie d'un bundle Astro pour améliorer les performances du site. La visualisation du bundle peut donner des indices sur les modifications à apporter à votre projet pour réduire la taille du bundle.
```
I still looked for what other documentation did, and for example React in French leaves the word "bundle". So I think we don't have to translate it.